project1-sps/script.js

```diff
@@ -1,4 +1,59 @@
+// HELPER FUNCTION to randomly generate number 1, 2 or 3
+var generateNumber = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomNumber = randomInteger + 1;
+  return randomNumber;
+};
+// END OF HELPER FUNCTION
+var currentGameMode = "inputName";
+var userName = "";
+var numUserWins = 0;
+var numComputerWins = 0;
+var numDraws = 0;
```

Comment:
Just nitpicking here, but declaring your global variables at the top of your script is best practice. This is so that anybody reading your code will be able to tell right off the bat what global variables you're using and also so that its easier for you to find!

Reply:
Thank you for the advice!